attribute_normalizer.gemspec

```diff
@@ -6,7 +6,7 @@ Gem::Specification.new do |s|
   s.name = %q{attribute_normalizer}
   s.version =AttributeNormalizer::VERSION
 
-  s.required_rubygems_version = Gem::Requirement.new("> 1.3.1") if s.respond_to? :required_rubygems_version=
+  s.required_rubygems_version = Gem::Requirement.new(">= 2.3.8") if s.respond_to? :required_rubygems_version=
```

Comment:
This deviates slightly from what the ticket specifies(`s.required_ruby_version = '>= 2.3.8'`) but I opted for minimal changes and left their implementation.

Reply:
Weird, yeah that's fine though